src/components/KonnectorTile.jsx

```diff
@@ -70,33 +43,45 @@ export class KonnectorTile extends Component {
       t
     } = this.props
     const { domain, secure } = this.context
-    const hideKonnectorErrors = flag('hide_konnector_errors')
+
+    let status
+
+    if (konnector.available_version) status = STATUS.UPDATE
+    else if (isInMaintenance) status = STATUS.MAINTENANCE
+    else if (error || userError) status = STATUS.ERROR
+    else if (!accountsCount) status = STATUS.NO_ACCOUNT
+    else status = STATUS.OK
+
     return (
       <NavLink
-        className="item-wrapper"
+        className={classNames('item-wrapper', {
+          '--ghost': status === STATUS.NO_ACCOUNT,
+          '--maintenance': status === STATUS.MAINTENANCE
```

Comment:
Nit: We should use item-wrapper--ghost and item-wrapper--maintenance to follow our existing class names. Maybe item-wrapper should be named item if it is too long to type ;)

Also, nit, why is the home not using css modules ?